pyiron_atomistics/sphinx/output_parser.py

```diff
@@ -404,8 +404,6 @@ def __init__(self, file_name="waves.sxb", cwd="."):
             file_name (str): file name
             cwd (str): directory path
         """
-        if cwd is None:
-            cwd = "."
         path = Path(cwd) / Path(file_name)
```

Comment:
This creates a problem if the filename is an absolute path.
I suggest something like

```
__init__(... cwd=None)
path = (cwd is not None) ? Path(cwd) / Path(file_name) : file_name

```
if you feel you need the cwd (I see the convenience). Wouldn't call it cwd though, because typical use cases will not be "current working directory" (which would be superfluous to add), but job directory.

Reply:
Agree, even if `cwd` is given the code should check whether `file_name` is absolute.  See `Path.is_absolute()`.

Reply:
Agree. So now I think the code checks, if the `file_name` is an absolute path and ignores cwd if True.

Reply:
That's a good point. Since `cwd` is always set in pyiron it won't be an urgent problem but I have to change this elsewhere as well.